nemo/collections/tts/modules/aligner.py

```diff
@@ -175,3 +180,56 @@ def forward(self, queries, keys, mask=None, attn_prior=None, conditioning=None):
 
         attn = self.softmax(attn)  # softmax along T2
         return attn, attn_logprob
+
+
+class AlignmentEncoderAdapter(AlignmentEncoder, adapter_mixins.AdapterModuleMixin):
```

Comment:
Do you think it would look more organized if we had an adapters.py with all available adapter modules?

Reply:
Yes. We can do that. Then where should we import `adapters.py` since we should register the adapter class in a global way.

Reply:
Add `modules/adapters.py`.